configure.ac

```diff
@@ -626,6 +627,12 @@ AC_ARG_ENABLE([util-util],
   [build_bitcoin_util=$enableval],
   [build_bitcoin_util=$build_bitcoin_utils])
 
+AC_ARG_ENABLE([experimental-util-chainstate],
+  [AS_HELP_STRING([--enable-experimental-util-chainstate],
```

Comment:
re: https://github.com/bitcoin/bitcoin/pull/24304#discussion_r808803775

> You may want to document this in the PR description and elsewhere.
> 
> Also should be in the `./configure` summary.

I believe this is only temporarily built as part of our build system and later moves to an examples folder (and  built with cmake!) So that may a reason not to add it to `--configure` help, though I guess it you could have a comment saying the option is temporary and will be removed later. Agree it makes sense to document in PR description and elsewhere (maybe at the top of `bitcoin-chainstate.cpp`) so it's obvious how to build & test this

Reply:
> So that may a reason not to add it to --configure help,

This option is already included in `./configure --help` output, that's done automatically by autotools.

> Also should be in the ./configure summary.

I don't think it needs to be.